test/fixtures.ts

```diff
@@ -0,0 +1,59 @@
+import type { PathLike } from 'fs';
+import callsite from 'callsite';
+import { DirectoryJSON, IFs, fs, vol } from 'memfs';
+import upath from 'upath';
+
+export class Fixtures {
+  static get(name: string): string {
+    const realFs = jest.requireActual('fs');
+    return realFs.readFileSync(upath.resolve(Fixtures.pathToFixtures, name), {
+      encoding: 'utf-8',
+    });
+  }
+
+  static mock(json: DirectoryJSON, cwd?: string): void {
```

Comment:
Maybe, switch argument order would be a good idea. At some point, we may want to extend this class to handle HTTP mocks as well. In this case, value of the first argument may be `/tmp/foo/bar` or `http://example.com/foo/bar`, while the second one defines content mapping for sub-path.

I didn't think a lot about how this API may look like exactly, it's more like just my random thought for now.

Reply:
Basically it is just proxy of `vol.fromJSON` from `memfs`, therefore order of arguments also just copied. Also second argument is optional, therefore swap of order of arguments is not possible.

Reply:
Okay, I see. Though, maybe not impossible, but is more complicated then just argument swap.